### DISCUSSION TITLE
How to automatically filter out "Live" albums/tracks?

### DISCUSSION THREAD

**Discussion by FadingArabChristians** [RandomNinjaAtk/docker-lidarr-extended#41] [General]
Is there a variable in the docker to filter out any "live"  albums?

**Comment by Protektor-Desura** [comment]
You tell Lidarr that you don't want any "live" albums.
Go to settings, profiles, then on the page go to metadata profiles and edit the profile you want and uncheck live albums.

**Reply by FadingArabChristians** [reply]
Thanks! 

I also noticed that the variable -e 'configureLidarrWithOptimalSettings' might be re-selecting "live" whenever there are rebuilds/upgrades to the docker

**Reply by RandomNinjaAtk** [reply]
Unless you've purged the local log files, it should only technically modify the lidarr settings one time... However, its a configurable option, you can just disable it...
